medcat/tokenizers/transformers_ner.py

```diff
@@ -92,6 +95,7 @@ def encode(self, examples: Dict, ignore_subwords: bool = False) -> Dict:
                         labels.append(self.label_map['X'])
 
                 if len(input_ids) >= self.max_len:
+                    logger.debug(f"Document exceeding max length encountered. Length of current document is {len(tokens['offset_mapping'])}. Performing chunking...")
```

Comment:
When logging, we generally want to use lazy evaluation.
That is, we want to pass an unformatted string with the `%s` (or similar) placeholders along with the relevant data / arguments.
This is because this way the (potentially expensive) task of formatting the string is only done when the message is going to be logged. Otherwise the formatting is done eagerly every time, regardless of whether or not it'll be shown anywhere. Thus the lazy evaluation helps save overhead in a non-debug situation.

While this would probably not be a huge issue in this specific case, we want to be consistent across the library.

So what we want here instead would be:
```
logger.debug("Document exceeding max length encountered. Length of current document is %d. Performing chunking...", len(tokens['offset_mapping']))
```

Reply:
That makes sense, I've made the change!